addon/templates/components/frost-object-browser.hbs

```diff
@@ -4,7 +4,9 @@
       {{title}}
     </div>
     {{# if subtitle}}
-     <div class='sub-title'>{{computedValuesTotal}} - {{subtitle}}</div>
+     <div class='sub-title'>
+         {{#if showCountInSubTitle}}{{computedValuesTotal}} -{{/if}} {{subtitle}}
```

Comment:
I think it is generally bad practice to use helpers in templates for conditionals as it is hard to test. Instead I would use a computed property like so.

**Component:**
```javascript
@readOnly
@computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
computedSubtitle (subtitle, showCountInSubTitle, count) {
  return showCountInSubTitle ? `${count} – ${subtitle}` : subtitle
}
```

**Template:**
```handlebars
<div class="sub-title">
  {{computedSubtitle}}
</div>
```

Then you can write unit tests against this computed property given different values for the three properties it is computed off of.